docs/src/main/asciidoc/spring-integration-support.adoc

```diff
@@ -10,6 +10,14 @@ Provide Spring Integration support for these Azure services: Event Hubs, Service
 
 === Spring Integration with Azure Event Hubs
 
+==== Key concepts
+Azure Event Hubs is a big data streaming platform and event ingestion service. It can receive and process millions of events per second. Data sent to an event hub can be transformed and stored by using any real-time analytics provider or batching/storage adapters.
+
+Spring Integration enables lightweight messaging within Spring-based applications and supports integration with external systems via declarative adapters. Those adapters provide a higher-level of abstraction over Spring’s support for remoting, messaging, and scheduling. The *Spring Integration for Event Hubs* extension project provides inbound and outbound channel adapters and gateways for Azure Event Hubs.
+
+Note: RxJava support APIs is dropped from version 4.0.0.
```

Comment:
Use `NOTE:` instead of `Note:`, because the rendered result is different. `NOTE:` is a keyword.

![image](https://user-images.githubusercontent.com/13167207/145498387-08c96866-77de-4768-bf26-391141904bce.png)

![image](https://user-images.githubusercontent.com/13167207/145498367-ac3c4fef-3a2d-40c6-9868-60c31c955aee.png)